### ISSUE TITLE
Major Nether 

### ISSUE THREAD

**Issue by Zelioth** [HaydenBelanger/assemblylinemachines#80]
I am the developer and creator of a pack called "Friends United" I have included ALM in my pack since it was released on 1.19.2. There seems to be a major issue with Generation in the nether where as there is no limit set to what I believe to be ore generation for ALM. This issue is not only causing massive log spam error in both single player but more noticeably in server but it is also using up a little bit of extra memory to what I believe is trying to correct this issue. out of 400 mods I have went through each one in sets and came to a group where ALM was included and put that group in a mod pack of it's own and disabled e
[2022-10-09-2.log](https://github.com/HaydenBelanger/assemblylinemachines/files/9740812/2022-10-09-2.log)
ach mod one by one until the error went away. This error only appeared and disappeared when ALM was enabled or disabled. I hope this helps with future updates although it may not be a major issue it does make including it in a server very impossible due to it's insane amount of spam anytime anyone visits the nether. here is the log with proof of error "[09Oct2022 00:18:27.316] [Worker-Main-7/WARN] [net.minecraft.world.level.levelgen.heightproviders.BiasedToBottomHeight/]: Empty height range: biased[0 above bottom--16 absolute inner: 1]"

There were over 1500 lines of this error in less that 5 minutes. I had to cut a large amount out due to size restriction of file

**Comment by Aantara** [comment]
I encountered the same issue, except it crashes my server whenever someone tries to enter the Nether due to the no height limit error spam. I've been troubleshooting this for the past 3 weeks trying to figure out which of my 200+ mods would affect the nether in such a way. As this is the only other mod in the pack that adds a dimension we hadn't been to yet, I checked here for similar issues, found this post, thought maybe I'd finally found my fix, removed the mod... and still have the issue.

**Comment by Zelioth** [comment]
Yeah as much as I love the mod I kicked it and don't plan on including it in my pack again. the 300+ players voted and said they were fine kicking it too since it was causing major TPS issues.  If you still have the issue it can actually be super easy to find the mod at fault. you aren't just looking for Biome modifications, you are also looking for ore generation. Come to our discord 

https://discord.gg/VHnkCSFHkw

I might know a few other mods that are at fault for this that still haven't fixed the issue

**Comment by EMyGuy** [comment]
Can confirm exact same issue on my 1.18.2 pack. I would be devastated to remove the mod, but the server crashing when entering the nether is kind of a big deal. Really hoping this gets fixed!

**Comment by no-name-provided** [comment]
I experienced this issue with the most recent 1.19.3 release. No significant performance hit, but silly logfile sizes. Interestingly, the message only seems to trigger when generating new chunks. Disabling black granite generation had no effect.

Installing logbegone and putting the following text in "/config/logbegone.toml" seems to be a temporary workaround:


[logbegone]
	phrases = ["the_aether", "Empty height range"]
	regex = [""]
-
It filters out the spam before it hits my logfile, and black granite generates as expected.
